cocotb/share/lib/embed/gpi_embed.c

```diff
@@ -259,52 +259,47 @@ int embed_sim_init(gpi_sim_info_t *info)
     }
 
     PyObject *cocotb_module, *cocotb_init, *cocotb_retval;
-    PyObject *simlog_obj, *simlog_func;
+    PyObject *cocotb_log_module = NULL;
+    PyObject *simlog_func;
     PyObject *argv_list;
 
     cocotb_module = NULL;
-    simlog_obj = NULL;
 
     // Ensure that the current thread is ready to call the Python C API
     PyGILState_STATE gstate = PyGILState_Ensure();
     to_python();
 
-    if (get_module_ref(COCOTB_MODULE, &cocotb_module))
+    if (get_module_ref("cocotb", &cocotb_module))
```

Comment:
What's the motivation for this change? For consistency with `cocotb.log` below? If so, please remove the definition of COCOTB_MODULE.

Reply:
Seemed like a fairly useless constant given its defined as a shorter string than its name. I'll come back and remove the env bar.